src/game/Entities/Player.cpp

```diff
@@ -1001,8 +1001,6 @@ uint32 Player::EnvironmentalDamage(EnviromentalDamage type, uint32 damage)
     damage = (damage <= malus ? 0 : (damage - malus));
 
     DamageEffectType damageType = SELF_DAMAGE;
-    if (type == DAMAGE_FALL && getClass() == CLASS_ROGUE)
```

Comment:
Why are you removing this? This breaks rogues falling and not breaking stealth while they have the passive ability.